starter-code/public/index.html

```diff
@@ -79,18 +79,19 @@
     <footer>
       &copy; 2017 Code Fellows | Static Blog | Happy Ipsum!
     </footer>
-    <!-- TODO: Be sure to include the routing functionality with the correct script files.
+    <!--Done! TODO: Be sure to include the routing functionality with the correct script files.
                Don't forget that the order of your scripts matters - variables have to be
                declared before you can reference them! -->
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="https://cdn.rawgit.com/visionmedia/page.js/master/page.js"></script>
     <script src="/vendor/scripts/handlebars.js"></script>
     <script src="/vendor/scripts/marked.js"></script>
     <script src="vendor/scripts/highlight.pack.js"></script>
     <script src="/scripts/models/article.js"></script>
     <script src="/scripts/views/articleView.js"></script>
     <script src="/scripts/controllers/articleController.js"></script>
-
+    <script src="./scripts/controllers/routes.js"></script>
```

Comment:
This is in the wrong place. When about is clicked, it then references the aboutController. But the about controller is below the routes, so it won't be able to find it. So your about page is not working. I get a can not GET error.